code/script.js

```diff
@@ -1,51 +1,82 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat");
+const input = document.getElementById("name-input");
+const sendBtn = document.querySelector(".send-btn");
+const nameForm = document.getElementById("name-form");
+const startImg = document.querySelector(".start-img");
+const startPage = document.querySelector(".start-page");
+const main = document.querySelector(".main");
 
```

Comment:
I found there is an overlapping sounds when the jazz, modern, and techno buttons are pressed, To fix the issue and to prevent the sound from triggering the input button in the main chatbot, you can adjust your JavaScript code as follows:  Create a Global Audio Object, Modify the Sound Functions:, Stop Audio When Necessary, Adjust Event Listeners  

// Create a global audio object
const audio = new Audio();

// ...

// Modify the sound functions to use the global audio object
const startMusic = (music) => {
  // Pause the current audio if it's playing
  audio.pause();

  // Set the new audio source
  audio.src = music;

  // Set other audio attributes (e.g., volume)
  audio.loop = true;
  audio.volume = 0.1;

  // Play the new audio
  audio.play();
};

// ...

// Adjust event listeners for the sound buttons to use the modified functions
document.querySelector(".music-container .jazz").addEventListener("click", () => {
  clickSound(event);
  startMusic("./assets/audio/piano.mp3");
});

document.querySelector(".music-container .chill").addEventListener("click", () => {
  clickSound(event);
  startMusic("./assets/audio/classic.mp3");
});

document.querySelector(".music-container .techno").addEventListener("click", () => {
  clickSound(event);
  startMusic("./assets/audio/techno.mp3");
});

// ...

// Create a function to stop the audio when necessary
const stopAudio = () => {
  audio.pause();
};

// Call the stopAudio function when appropriate, such as when switching between different audio options or exiting the chatbot
// Example usage:
// stopAudio();

Reply:
Thank you for letting me know that there is a bug. I think I fixed it.